### ISSUE TITLE
Clean up browser creation & destruction

### ISSUE THREAD

**Issue by jfinkhaeuser** [spriteCloud/lapis-lazuli#14]
- Check defaults when using `browser.create`. It currently uses configuration defaults, but IMHO should use the same configuration as when using `browser` directly, with the ability to override.
- Ensure browsers created by `browser.create` also get destroyed.

**Comment by jfinkhaeuser** [comment]
Mostly fixed in (upcoming) 0.7 release. Review of interest.

**Comment by barzilay** [comment]
Fixed in ll 1.0. Closing after end is not possible because connection to the browser is already lost. Closing happens just before that happens.

**Comment by sjieg** [comment]
```
# Making sure all browsers are gracefully closed when the exit event is triggered.
at_exit { LapisLazuli::Browser::close_all 'exit event trigger' }
```
this piece of code ensured a positive destruction.
